### DISCUSSION TITLE
How to excute external command, like `:!ls`

### DISCUSSION THREAD

**Discussion by LintaoAmons** [folke/noice.nvim#333] [Q&A] [answered]
I wonder how to execute an external shell command by Noice?

Before I could use `!` before the command to indicate vim run it as an external shell command.

But it seem that after I type `!` in Noice, it switch to filter instead execute the as a shell command.

**Comment by folke** [comment, marked as answer]
`!` is filter and is the same as running an external command? So what do you mean? Did you actually try running something like `!ls`?

**Reply by LintaoAmons** [reply]
I didn't, saw the title changed to filter after type `!`, then didn't continue...
